### ISSUE TITLE
Update documentation regarding "community" Slack

### ISSUE THREAD

**Issue by lindhe** [harness/harness#3359]
This line should be updated:

https://github.com/harness/drone/blob/e63b8f9326969220df73b9d74349405eb86c1073/.github/readme.md?plain=1#L21

The Slack is clearly not open for the community, since one must have a @harness.io address to join:

![Slack enforces users to have a @harness.io address](https://github.com/harness/drone/assets/7773090/6e77e4ad-5cb3-41a4-9965-b6c46d766b27)

**Comment by wez** [comment]
is drone.io as a self-hosted solution still alive and well? Where are open source users supposed to go to discuss using drone? The slack is not open, and the former forums are now read-only.
Should I be looking for an alternative self-hosted CI solution?

**Comment by wojtekLs** [comment]
> Should I be looking for an alternative self-hosted CI solution?

@wez very much so. At least thats what Im going to do. `Drone` was ok till it was independent. Its greatness has been declining ever since they had been bought-out by Harness.

**Comment by wez** [comment]
https://woodpecker-ci.org seems like a reasonable next stop from drone

**Comment by wojtekLs** [comment]
@wez not at all. `Woodpecker` is just `drone`, just with community additions.

**Comment by shubhalgupta** [comment]
@lindhe Kindly assign this issue to me!

**Comment by lindhe** [comment]
@shubhalgupta How do I do that?

**Comment by shubhalgupta** [comment]
@lindhe in assignees u have to add my name

**Comment by lindhe** [comment]
@shubhalgupta I cannot:

![image](https://github.com/harness/gitness/assets/7773090/a426857a-ab85-4491-ab53-c5eb47fa4acf)


Guess I don't have permissions to do so.

**Comment by lindhe** [comment]
I don't know, it's up to you. I know that what it says today is incorrect, I don't know what you intend it to say.

**Comment by shubhalgupta** [comment]
@lindhe should I update it to Join Harness community through your Harness email? And remove the link to join slack?

**Comment by bradrydzewski** [comment]
we are handling this.  someone internally changed the Slack settings to require a harness.io email address.  I have asked the team to revert this change by end of day.   I will post and close this issue once resolved.
